images/kwok/build.sh

```diff
@@ -139,15 +144,29 @@ function main() {
       "--platform=${platform}"
     )
   done
-  if [[ "${PUSH}" == "true" ]]; then
-    extra_args+=("--push")
+
+  if [[ "${BUILDER}" == "docker" ]]; then
+    if [[ "${PUSH}" == "true" ]]; then
+      extra_args+=("--push")
+    else
+      extra_args+=("--load")
+    fi
+    dry_run docker buildx build \
+      "${extra_args[@]}" \
+      -f "${DOCKERFILE}" \
+      .
+  elif [[ "${BUILDER}" == "nerdctl" ]]; then
+    dry_run nerdctl build \
+            "${extra_args[@]}" \
+            -f "${DOCKERFILE}" \
+            .
+    if [[ "${PUSH}" == "true" ]]; then
+      nerdctl push ${image}:${tag}
```

Comment:
Will this push multi architecture image?

Reply:
an args can support all platforms, let me add it here